src/secrets/components/secrets.scss

```diff
@@ -0,0 +1,13 @@
+.secrets-context-menu {
+  padding-right: 15px;
+}
+.warning-panel-icon {
+  font-size: 24px;
+  background: -webkit-linear-gradient(
+    45deg,
+    rgb(232, 91, 28) 0%,
+    rgb(255, 185, 74) 100%
+  );
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
```

Comment:
we should use `cf` variables whenever possible to make sure that styling stays consistent